elementary/monitor/cli.py

```diff
@@ -15,6 +15,15 @@
 
 # Displayed in reverse order in --help.
 def common_options(func):
+    func = click.option(
+        "--dbt-quoting",
```

Comment:
In continue to my comments, we could except here for --env-vars which will be a string of a json/yaml of the env vars which we will load as dict and pass it to the command runner :)
We could also support a .env file path that will be loaded as the env vars. WDYT?

Reply:
Added a generic param in addition to the quoting one, as discussed on Slack